### DISCUSSION TITLE
[Feature Request] Link for Creating New Codespace or Open Existing One

### DISCUSSION THREAD

**Discussion by marchof** [community/community#37313] [Codespaces]
Hi GitHub Team,

the documentation describes [how to add a badge](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-codespaces-badge) to a repository for quickly starting an codespace. This is a powerful feature as it allows organisation users to quickly get started with development. Unfortunately when using the badge link it will always create a new workspace.

It would be a nice option, if the link behind the button would list existing workspaces of that user for the workspace which could be directly opened.

I'm aware that existing workspaces can be seen under the green code button, but this makes the badge kind of useless if it can only be used to create a new workspace.

Cheers,
-marc

**Comment by ossanna16** [comment]
Hi there @marchof and welcome to our community! Thank you for your feature request 🙂

**Comment by marchof** [comment]
Answering my own feature request: There is actually already a link which shows the exiting spaces for a given project and also allows to create a new space. The only limitation is that the space cannot be preconfigured:

https://github.com/organization/project/codespaces
